<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Principal</class>
 <widget class="QWidget" name="Principal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Principal</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="2">
    <widget class="Frame" name="frame2" native="true"/>
   </item>
   <item row="2" column="1">
    <widget class="Frame" name="frame4" native="true"/>
   </item>
   <item row="1" column="3">
    <widget class="Frame" name="frame3" native="true"/>
   </item>
   <item row="2" column="2">
    <widget class="Frame" name="frame5" native="true"/>
   </item>
   <item row="1" column="1">
    <widget class="Frame" name="frame1" native="true"/>
   </item>
   <item row="2" column="3">
    <widget class="Frame" name="frame6" native="true"/>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="cbSeleccionarCamara"/>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="pbIniciarCamara">
     <property name="text">
      <string>Iniciar c√°mara</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Frame</class>
   <extends>QWidget</extends>
   <header location="global">frame.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
